### DISCUSSION TITLE
How to implement subtitle post processing script

### DISCUSSION THREAD

**Discussion by Fuergrissa1** [pymedusa/Medusa#9790] [Q&A]
My media player (Plex) crashes when I play media with external SSA/ASS subtitle files. I would like to run a script after downloading subtitles in Medusa to convert any SSA/ASS files to SRT.

I found this project: [https://github.com/sorz/asstosrt](url) that sounds like it will do what I want but I'm a complete noob and can't figure out how to install or configure it.

Could someone please take a look and help me with some more idiotproof instructions?

**Comment by 007revad** [comment]
Can you answer the following questions before I jump down the rabbit hole of writing a python script and explaining how set Medusa to use.

Are you still looking for an answer to this?

In my experience Medusa only downloads SRT subtitles. Which subtitle plugins do you have enabled?

Or maybe you have Medusa set to keep .SSA and/or .AAS files in "Post-Processing > Keep associated file extensions"?

**Comment by Fuergrissa1** [comment]
Don't worry about it. I do still have the problem but it doesn't effect me every day. I thought it would be easier to setup.
